### PR TITLE
add email message

### PR DESCRIPTION
**Pull request by gedankenstuecke**

### CODE REVIEW

**Comment by gedankenstuecke** [comment]
Ok, I think this is the MVP for transferring the 23andMe processing over to the stand-alone project. 
@madprime given that you're most familiar with the process: Could you look over it at some point?

File: main/celery.py [hunk 1]
```diff
@@ -215,90 +218,102 @@ def clean_raw_23andme(closed_input_file):
...
+
+        upload_new_file(vcf_filename,
                         access_token,
                         str(member['project_member_id']),
                         metadata)
-    api.delete_file(access_token,
-                    str(member['project_member_id']),
-                    file_id=str(dfile['id']))
+    except:
+        raise
```
Comment (madprime) [inline main/celery.py]: I think this needs to happen *after* calling `api.message` :)

**Review by madprime — COMMENTED** [review]
Good to merge once this is fixed I think! There's some other issues, e.g. with zipfile, but let's do those in other PRs... :smile:

File: main/celery.py (hunk 1 above)
Comment (gedankenstuecke) [reply main/celery.py]: yep, you're right, reordered that

**Pull request merged**